bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/member/AmendMembershipMetadataFlow.kt

```diff
@@ -18,21 +19,24 @@ data class AmendMembershipMetadataRequest(val metadata : Any)
  * Proposes a change to the membership metadata
  */
 @InitiatingFlow
-class AmendMembershipMetadataFlow(private val newMetadata : Any) : FlowLogic<SignedTransaction>() {
+class AmendMembershipMetadataFlow(bno : Party, private val newMetadata : Any) : BusinessNetworkAwareInitiatingFlow<SignedTransaction>(bno) {
```

Comment:
Is it possible to ensure that the bno parameter at this point is actually BNO? If so it should probably be done.

Reply:
It is done inside `BusinessNetworkAwareInitiatingFlow`